apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -75,16 +91,21 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
     } = props;
 
     const { t } = useTranslation();
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
     const { isMobileViewport } = useMediaContext();
 
     const profileDetails: AuthStateInterface = useSelector((state: AppState) => state.authenticationInformation);
     const isProfileInfoLoading: boolean = useSelector((state: AppState) => state.loaders.isProfileInfoLoading);
     const isSCIMEnabled: boolean = useSelector((state: AppState) => state.profile.isSCIMEnabled);
     const profileSchemaLoader: boolean = useSelector((state: AppState) => state.loaders.isProfileSchemaLoading);
-    const isReadOnlyUser = useSelector((state: AppState) =>
+    const isReadOnlyUser: string = useSelector((state: AppState) =>
         state.authenticationInformation.profileInfo.isReadOnly);
-    const config = useSelector((state: AppState) => state.config);
+    const config: ConfigReducerStateInterface = ...
+        DeploymentConfigInterface,
```

Comment:
There should not be a line break here, right?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3763/commits/c1a35d02e973f3b654488bf31c6dfa16fb055617